ra/registration-authority.go

```
@@ -225,6 +228,6 @@ func (ra *RegistrationAuthorityImpl) OnValidationUpdate(authz core.Authorization
 		authz.Expires = time.Now().Add(365 * 24 * time.Hour)
 	}
 
-	// Finalize the authorization
-	ra.SA.FinalizeAuthorization(authz)
+	// Finalize the authorization (error ignored)
+	_ = ra.SA.FinalizeAuthorization(authz)
```

Why are we ignoring the error here rather than passing it up?